databases/redis-enterprise-operator/manifests/04-ssl/client-pod-tls.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: redis-client-d
+spec:
+  containers:
+  - name: redis-client-d
+    image: redis:latest
+    command: ["sleep", "3600"]
+    env:
+    - name: PORT 
+      valueFrom:
+        secretKeyRef:
+          name: redb-d-app-db           
+          key: port
+    - name: PASS
+      valueFrom:
+        secretKeyRef:
+          name: redb-d-app-db
+          key: password
+    - name: SERVICE
+      valueFrom:
+        secretKeyRef:
+          name: redb-d-app-db
```

Comment:
this should be redb-app-db , otherwise the pod will be stuck searching for a non-existent secret